website/content/prometheus-statistics/_index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Prometheus statistics
```

Comment:
nit: any reason why you called this "statistics" and not "metrics"? I think "metrics" is a bit more natural when it comes to prometheus

Reply:
+1

Reply:
No specific reason, I just saw an example of such a title in another project and went with it.
I agree with you that metrics fit better.